docs/src/components/Layout/Header.tsx

```diff
@@ -33,6 +42,14 @@ export const Header = ({
         src={`/svg/integrations/${platform}.svg`}
       />
 
+      {showSearch && (
+        <DocSearch
+          appId={process.env.DOCSEARCH_DOCS_APP_ID}
+          apiKey={process.env.DOCSEARCH_DOCS_API_KEY}
+          indexName={process.env.DOCSEARCH_DOCS_INDEX_NAME}
+        />
+      )}
+
```

Comment:
Right now on every page except the homepage, the search gets hidden by the sidebar. The quick fix is to move this into the `<Flex>` component on line 53.